backend/medtagger/database/models.py

```diff
@@ -426,3 +426,32 @@ def __init__(self, slice_index: int, width: int, height: int, tag: LabelTag) ->
     def __repr__(self) -> str:
         """Return string representation for Brush Label Element."""
         return '<{}: {}>'.format(self.__class__.__name__, self.id)
+
+
+class PointLabelElement(LabelElement):
+    """Definition of a Label Element made with Point Tool."""
+
+    __tablename__ = 'PointLabelElements'
+    id: LabelElementID = Column(String, ForeignKey('LabelElements.id'), primary_key=True)
+
+    x: float = Column(Integer, nullable=False)
```

Comment:
This should be a Float Column (not Integer). Typing is fine but DB model is wrong :)